config-connector/tests/testcases/environments.template

```diff
@@ -0,0 +1,2 @@
+DEFAULT_PROJECT_ID: ${DEFAULT_PROJECT_ID?}
```

Comment:
Should it be PROJECT_ID for consistency?

Reply:
I think we want the env var names to be more descriptive, especially for the ones that is not unique. There will be NEW_PROJECT_ID, SHARED_VPC_SERVICE_PROJECT_ID, SHARED_VPC_HOST_PROJECT_ID, etc in the future in the environments.template. So I decided to add an attributive for all the project ID env vars.